packages/carousel/src/Carousel.tsx

```diff
@@ -28,182 +28,129 @@ interface Props extends CalculatedProps {
   disableScroll: boolean;
 }
 
-interface State {
-  slideIndex: number;
-  swiping: boolean;
-}
-
-export class Carousel extends Component<Props, State> {
-  static defaultProps = {
-    items: [],
-    margin: 0,
-    distanceBetweenItems: 0,
-    arrowOffset: 0,
-    buttonClass: '',
-    wrapperClass: '',
-  };
-
-  swipeDistance = 0;
-  swipeSpeed = 0;
-  slideshowRef = React.createRef<HTMLDivElement>();
-
-  state: State = {
-    slideIndex: 0,
-    swiping: false,
-  };
-
-  // Todo: All slides ar left aligned when the (number of) slides does not fill
-  // the entire screen. Can be reproduced by limiting the number of slides in the carousel
-  static getDerivedStateFromProps(nextProps: Props, prevState: State) {
-    const { columnsPrSlide, items } = nextProps;
-    const { slideIndex } = prevState;
-
-    // Check if resize caused carousel to be scrolled to far.
-    if (Math.floor(columnsPrSlide) - items.length > slideIndex) {
-      const adjustedSlideIndex = slideIndex + (Math.floor(columnsPrSlide) - items.length - slideIndex);
-
-      if (items.length < columnsPrSlide) {
-        return {
-          slideIndex: 0,
-        };
+export const Carousel: React.FC<Props> = ({
+  items = [],
+  columnWidth,
+  columnsPrSlide,
+  slideBackwardsLabel,
+  slideForwardsLabel,
+  disableScroll,
+  arrowOffset = 0,
+  distanceBetweenItems = 0,
+  margin = 0,
+  buttonClass = '',
+  wrapperClass = '',
+}) => {
```

Comment:
Synes denne notasjonen er ryddigere:
```suggestion
export const Carousel = ({
  items = [],
  columnWidth,
  columnsPrSlide,
  slideBackwardsLabel,
  slideForwardsLabel,
  disableScroll,
  arrowOffset = 0,
  distanceBetweenItems = 0,
  margin = 0,
  buttonClass = '',
  wrapperClass = '',
}: Props) => {
```

Reply:
Enig.